src/interface/report/hooks/useEvents.ts

```diff
@@ -23,53 +16,47 @@ const useEvents = ({
   player: PlayerInfo;
 }) => {
   const [events, setEvents] = useState<AnyEvent[] | null>(null);
-  // we manually paginate here. WCL could turn pagination back on in the WCL API but the amount loaded per page is too low
-  // for M+ to be usable without deeper changes on the WCL side.
-  const pageCount = fightPageCount(fight);
-  const [pagesLoaded, setPagesLoaded] = useState(0);
+  const [currentTime, setCurrentTime] = useState<number>(fight.start_time);
 
   useEffect(() => {
```

Comment:
this reminds me that I want to refactor our data loading to not use `useEffect` directly at some point. I'll probably lump that into my to-do list.

Reply:
Made an issue for it here: https://github.com/WoWAnalyzer/WoWAnalyzer/issues/6894